packages/reports/tests/acceptance/column-config-test.js

```diff
@@ -1364,6 +1348,9 @@ module('Acceptance | Navi Report | Column Config', function(hooks) {
       ['Date Time (Day)', 'Platform Revenue (CAD)'],
       'Clicking the filtered option changes the metrics parameter'
     );
+
+    await click(findAll('.grouped-list__group-header').filter(el => el.textContent.includes('Revenue'))[0]);
+    assert.dom('.metric-config').doesNotExist();
```

Comment:
Can we add an assert message?
```suggestion
    assert.dom('.metric-config').doesNotExist('Metric config is not visible when request preview is enabled');
```